### PR TITLE
Issue 4513 - Fix schema test and lib389 task module

### PR DESCRIPTION
**Pull request by droideck**
Description: Fix the asssertion in schema_test.py.
Make sure that all of the tasks are up to date
with the recent changes in the task API.

Relates: #4513

Reviewed by: ?

### CODE REVIEW

File: dirsrvtests/tests/suites/clu/schema_test.py [hunk 1]
```diff
@@ -52,12 +52,11 @@ def test_origins_with_extra_parenthesis(topo):
     attr_result = schema.query_attributetype(ATTR_NAME, json=True)
 
     # Verify the x-origin value is correct
-    assert attr_result['at']['x_origin'][0] == X_ORG_VAL
+    assert attr_result['at']['x_origin'][0][0] == X_ORG_VAL
```
Comment (mreynolds389) [inline dirsrvtests/tests/suites/clu/schema_test.py]: This just seems wrong to me.  I recently added this test case to test the fix for the embedded parenthesis, and to have this double nested subscription does not look right.  What does the dict look like to require this?  I'm just wondering if this is something that needs to be fixed in lib389 instead (maybe a regression was introduced in lib389)?

Comment (droideck) [reply dirsrvtests/tests/suites/clu/schema_test.py]: Right... I am not sure what made me do the fix like this. 
Please, check another fix :)

**Comment by Firstyear** [comment]
I think it looks fine to me now, but I'll let @mreynolds389 check to be sure.

**Review by mreynolds389 — APPROVED** [review]
LGTM

**Pull request merged**